samcli/lib/providers/sam_base_provider.py

```diff
@@ -161,7 +161,7 @@ def _extract_sam_function_imageuri(resource_properties: Dict, code_property_key:
         str
             Representing the local imageuri
         """
-        return resource_properties.get(code_property_key, None)
```

Comment:
Won't this default to `None` anyways 🤔 , wonder why mypy doesn't like it if its explicitly defined.

Reply:
Oh I've changed this line since calling `dict.get(key)` is equal to `dict.get(key, None)` 😅. The actual part that we needed to update was at line 148.